instructor/common.py

```diff
@@ -85,6 +86,28 @@ def create_new_assignment(request):
     return redirect('common_assignment_setup')
 
 
+def get_pages(assignment):
+    """
+    Return a dictionary of enabled/disabled pages
+    """
+    pages = {}
+    enabled = True
+    for page in page_order:
+        pages[page] = enabled
```

Comment:
After the case `page == assignment.last_page_name` I want enabled to stay False.